pkg/core/localize/locales/en/cmd/context.en.toml

```diff
@@ -233,6 +243,12 @@ The context doesn't have a Kafka instance ID.
 Use the "rhoas context set-kafka" command to select a Kafka instance for your context
 '''
 
+[context.common.error.noConnectorID]
+one='''
+The rhoas context doesn't have a connector instance ID.
+Use the "rhoas context set-connector" command to select a connector instance for your context
```

Comment:
The Connectors instance for the current context is not set. The `rhoas` CLI uses the current Connectors instance when you run any `rhoas connector cluster` commands.
Use the `rhoas context set-connector` command to set the current Connectors instance.